test/FastTests/Client/FirstClassPatch.cs

```diff
@@ -712,6 +712,29 @@ public async Task CanAddToArrayAsync()
                     Assert.Equal(loaded.Numbers[7], 202);
                 }
             }
+
+            async Task SaveChangesWithTryCatch(IAsyncDocumentSession session, User loaded)
```

Comment:
I know this is needed only for investigation but can we write it as helper method so we'll avoid repeating it multiple times. Once we figure out what is the problem it will be easier remove just a single method